android/anc/src/test/java/org/smartregister/fhircore/anc/data/family/FamilyRepositoryTest.kt

```diff
@@ -113,6 +114,35 @@ class FamilyRepositoryTest : RobolectricTest() {
     coVerify { resourceMapperExtended.saveParsedResource(any(), any(), any(), null) }
   }
 
+  @Test
+  fun updateProcessFamilyMemberShouldExtractEntities() = runBlockingTest {
+    val defaultRepository = spyk(DefaultRepository(fhirEngine))
+
+    val resourceMapperExtended = spyk(ResourceMapperExtended(defaultRepository))
+
+    coEvery { resourceMapperExtended.saveParsedResource(any(), any(), any(), "1111") } just runs
+
+    ReflectionHelpers.setField(repository, "resourceMapperExtended", resourceMapperExtended)
+
+    repository.updateProcessFamilyMember("1111", Questionnaire(), QuestionnaireResponse(), "1111")
+
+    coVerify { resourceMapperExtended.saveParsedResource(any(), any(), any(), "1111", eq(true)) }
```

Comment:
Verify that the Questionnaire() and QuestionnaireResponse() passed on line 127 are the same here